### PR TITLE
Sonos: play TuneIn by a station id

### PR DESCRIPTION
**Pull request by lolodomo**
Channel table fully updated in the documentation with the access mode added for each channel

Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

**Comment by lolodomo** [comment]
@kgoderis : could you please review this change ?

**Comment by sjka** [comment]
Just tried it out - works like a charm for me 👍

File: extensions/binding/org.eclipse.smarthome.binding.sonos/src/main/java/org/eclipse/smarthome/binding/sonos/handler/ZonePlayerHandler.java [hunk 1]
```diff
@@ -132,6 +134,8 @@
 
     private Map<String, String> stateMap = Collections.synchronizedMap(new HashMap<String, String>());
 
+    private List<SonosMusicService> msuicServices;
```
Comment (wborn) [inline extensions/binding/org.eclipse.smarthome.binding.sonos/src/main/java/org/eclipse/smarthome/binding/sonos/handler/ZonePlayerHandler.java]: I don't own or use any Sonos devices. But "msuicServices"  looks like a typo. :-)

Comment (kaikreuzer) [reply extensions/binding/org.eclipse.smarthome.binding.sonos/src/main/java/org/eclipse/smarthome/binding/sonos/handler/ZonePlayerHandler.java]: Good catch :-)
Ah, sorry, don't know where my head is these days... I somehow mixed up Sonos with LIFX... So forget about my testing request unless you can make your bulbs play some music 🤣

Comment (lolodomo) [reply extensions/binding/org.eclipse.smarthome.binding.sonos/src/main/java/org/eclipse/smarthome/binding/sonos/handler/ZonePlayerHandler.java]: lol

**Comment by lolodomo** [comment]
Typo fixed + everything squashed.

**Pull request merged**